packages/cra-template-mephisto-review/template/src/plugins/WordCloudRenderer.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { H4 } from "@blueprintjs/core";
+import WordCloud from "../components/WordCloud";
+
+/*
+    EXAMPLE PLUGIN ITEM RENDERER
+    Renders mephisto review data items as word clouds of the most common words in the object
+*/
+function WordCloudRenderer({ item }) {
+  if (!item) return <p>No Data Available</p>;
+  const data = item.data;
+  const id = item.id;
+  return (
+    <div>
+      <H4>
+        <b>ID: {id}</b>
+      </H4>
+      <H4>Data keywords:</H4>
+      <WordCloud
+        data={data}
+        filterKeys={["file"]}
+        filterWords={[
```

Comment:
i like excludeKeys and excludeWords better here. A lot of people get confused when using the JS `[].filter()` about whether it filters *out* or *in* based on how to boolean evaluates... I think excludes is more explicit and clear. For example, in this current usage, filterKeys is unclear whether it excludes that key or includes just that key.

Reply:
Definitely makes more sense! Will add.